sklearn/feature_extraction/text.py

```diff
@@ -607,9 +607,10 @@ class CountVectorizer(BaseEstimator, VectorizerMixin):
         A mapping of terms to feature indices.
 
     stop_words_ : set
-        Terms that were ignored because
-        they occurred in either too many
-        (`max_df`) or in too few (`min_df`) documents.
+        Terms that were ignored because they either
+        - occurred in too many documents (`max_df`)
+        - occurred in too few documents (`min_df`)
+        - were cut off by feature selection (`max_features`).
```

Comment:
Adding 2 spaces before `-` should fix it... Ref [`gradient_boosting.GradientBoostingClassifier` documentation](http://scikit-learn.org/stable/modules/generated/sklearn.ensemble.GradientBoostingClassifier.html?highlight=max_features) and [the corresponding source](https://github.com/scikit-learn/scikit-learn/blob/master/sklearn/ensemble/gradient_boosting.py#L1182).